Prevents:
Error: cannot open XML file ../src/help/C/index.docbook

--- a/meson.build
+++ b/meson.build
@@ -218,7 +218,6 @@ subdir('cut-n-paste')
 subdir('src')
 subdir('man')
 subdir('plugins')
-subdir('help')
 subdir('data')
 subdir('doc')
 
